### PR TITLE
Omnibus Toolchain support

### PR DESCRIPTION
**Pull request by schisamo**
/cc @opscode-cookbooks/ociv

### CODE REVIEW

File: recipes/_omnibus_toolchain.rb [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    group 'root'
+    mode '0444'
+  end
+
+  case node['kernel']['machine']
+  when 'i86pc'
+    package_url = 'https://chef-releng.s3.amazonaws.com/omnibus/omnibus-toolchain/omnibus-toolchain-0.0.1-1.i86pc.solaris'
+    package_checksum = 'eed0d5cc82c76dffd0052899a85ec96e3dc6915bdabc8d7c5c209028155744ea'
+  when 'sun4v', 'sun4u'
+    package_url = ''
```
Comment (schisamo) [inline recipes/_omnibus_toolchain.rb]: This will be updated once the SPARC package is ready.

File: recipes/_environment.rb [hunk 2]
```diff
@@ -133,6 +124,16 @@
     EOH
   end
 
+  if solaris_10?
+    omnibus_env['PATH'] << '/usr/local/bin'
+    omnibus_env['PATH'] << '/usr/ccs/bin'
```
Comment (scotthain) [inline recipes/_environment.rb]: Need to swap the order here, sfw first then ccs

File: recipes/_environment.rb [hunk 3]
```diff
@@ -133,6 +124,16 @@
...
+    omnibus_env['PATH'] << '/usr/sfw/bin'
+    # Solaris 10 supported make is gmake
+    additional_config = <<-EOH.gsub(/^ {6}/, '')
+      alias make='gmake'
```
Comment (scotthain) [inline recipes/_environment.rb]: Not needed, since we include make in the omnibus toolchain as 'make'

**Comment by yzl** [comment]
+1

**Comment by scotthain** [comment]
:shipit:

**Pull request merged**